docs/_static/custom-icons.js

```diff
@@ -0,0 +1,40 @@
+// This file is referenced in the docs, so if you make changes to it be sure to
+// update the docs too :)
```

Comment:
Since this file is not minified on our docs site, I assume it's not minified on theme-adopter sites. So I'm going to remove all unnecessary comments and just link to the docs page instead.

You can see it's not minified by visiting the URL: https://pydata-sphinx-theme.readthedocs.io/en/latest/_static/custom-icon.js?v=74687d30